examples/2d/mesh.rs

```diff
@@ -0,0 +1,148 @@
+use bevy::{
+    prelude::*,
+    render::{
+        pipeline::{PipelineDescriptor, RenderPipeline},
+        shader::{ShaderStage, ShaderStages},
+    },
+};
+
+fn main() {
+    App::build()
+        .insert_resource(WindowDescriptor {
+            title: "A Shaded Star".to_owned(),
+            width: 500.0,
+            height: 500.0,
+            ..Default::default()
+        })
+        .add_plugins(DefaultPlugins)
+        .add_startup_system(star.system())
+        .run();
+}
+
+fn star(
+    // Commands are used to spawn a new entity
```

Comment:
I think this might be too much commenting. Commands is fundamental and introduced in other examples/documentation. I think it's better to assume that anyone reading a more advanced example like this knows what it is. The same may be true for `meshes` but `pipelines` and `shaders` probably deserve comments.